addons/web/static/src/js/views/kanban/kanban_column_quick_create.js

```diff
@@ -45,8 +46,10 @@ var ColumnQuickCreate = Widget.extend({
         this.$quickCreateUnfolded = this.$('.o_quick_create_unfolded');
         this.$input = this.$('input');
 
-        // destroy the quick create when the user clicks outside
-        core.bus.on('click', this, this._onWindowClicked);
+        if (this.foldOnWidowClick) {
```

Comment:
typo. 
I wonder if a name like `isFirstColumn` wouldn't be clearer ?

Reply:
@SimonGenin I changed it to ````isFirstColumn```